.github/workflows/on_pull_request.yml

```diff
@@ -163,3 +163,35 @@ jobs:
       # - run: bundle exec danger --dangerfile=".ci/danger/apk_comparison.Dangerfile" --danger_id='apk_comparison' --remove-previous-comments
       #   env:
       #     DANGER_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  # TODO This is a temporary one to check the test. You can delete it.
+  # Emurator's test is not stable now. Sometimes fail..
+  instrumentation-test:
+    runs-on: macOS-latest
+    if: contains(github.event.pull_request.title, '[run instrumentation test]') == true
```

Comment:
Only works when `[run instrumentation test]` is in the title of the pull request.